### PR TITLE
Adding weight and bias support

### LINKED ISSUE #5338: Allow Object Attributes in AST
## Description
Allow for object attributes to be allowed in the allowlist AST.
The issue being that they do not exist at Class analysis time, and we would want the bound function to have the context of the target data item.

One method could be to provide a flag that these are dynamic and not bind the method but attach generated function to the Pointer so that on the retrieval side we can attempt to locate the attribute at execution time. 

## Definition of Done
Items like the following can be enabled and work as expected:
```
allowlist["torch.nn.Linear.weight"] = "torch.Tensor"
allowlist["torch.nn.Linear.bias"] = "torch.Tensor"
```

### PR DESCRIPTION
**Pull request by tudorcebere**
## Description
fixes #5338

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Review by madhavajay — APPROVED** [review]
So much cool!
